examples/getting_started/example_multiphase.py

```diff
@@ -124,6 +124,9 @@ def prepare_ocp(
 
     ocp = prepare_ocp(long_optim=True)
 
+    # --- Print ocp structure --- #
+    ocp.structure_graph(print_to_terminal=True)
```

Comment:
I think the method could be call `print_ocp_structure` or `print_ocp` and then the `print_to_terminal` is redundant (could now only be `to_terminal`, and I think Python calls it `console` instead of terminal)

Reply:
Done